lib/wab/impl/model.rb

```diff
@@ -174,12 +183,30 @@ def format_obj(format, ref, rid, obj)
         end
       end
 
+      def load_files()
+        unless @dir.nil?
```

Comment:
prefer using guard clauses instead of wrapping the entire body in conditional:
```ruby
def load_files
  return if @dir.nil?
  [...]
end
```

Reply:
Thats more of a style issue. I prefer the wrapper approach for smaller bits of code and 3 lines is a small bit.